kubernetes/cray-dhcp-kea/values.yaml

```diff
@@ -127,6 +128,7 @@ cray-service:
           port: 8000
         initialDelaySeconds: 30
         periodSeconds: 60
+        timeoutSeconds: 20
```

Comment:
This seems pretty long for a port check.   Won't cause a major issue, but will delay setting the pod to unready if the port goes away.

Reply:
adjusted timeout to be 5s.